pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/UnaryOperator.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.apex.ast;
+
+import apex.jorje.data.ast.PostfixOp;
+import apex.jorje.data.ast.PrefixOp;
+
+/**
+ * Apex unary operator
+ */
+public enum UnaryOperator {
```

Comment:
I suppose having an enum that merges prefix and postfix operators is only useful if we also merge the nodes ASTPrefixExpression and ASTPostfixExpression into a single ASTUnaryExpression... Maybe we can do that in PMD 7? I wonder otherwise why you would do that.

Reply:
I originally created only `BinaryOperator` and `UnaryOperator` to match the two types of operators in our AST library, but I agree that it'd be more appropriate to match the PMD wrapper nodes instead. Thanks for the feedback.